code/python_call_screening

```diff
@@ -0,0 +1,25 @@
+from flask import Flask, request
+from signalwire.voice_response import VoiceResponse
+
+app = Flask(__name__)
+blockList = ['+15551238104']
```

Comment:
Use an environment variable here please. You can use `split()` to convert a comma separated list into an array.